.github/workflows/release-prod.yml

```diff
@@ -6,12 +6,20 @@ on:
       - 'v*'
 
 jobs:
-  release:
+  check-master:
     runs-on: ubuntu-latest
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Check master branch
+        id: check
+        if: github.event.base_ref == 'refs/heads/master'
```

Comment:
I would like to confirm the value of `base_ref` attribute, as per this push payload example [https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#webhook-payload-example-35](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#webhook-payload-example-35l), `base_ref` can have `null` value as well.